### DISCUSSION TITLE
When I replace dist and the service is not restarted, it is still the last dist result when I refresh the access.

### DISCUSSION THREAD

**Discussion by kemengwang** [vikejs/vike#652] [Discussion]
### Description

I want to use this framework in the company's ssr construction.However, when I combine publishing and directly replace dist products without restarting the service, page access is still the result of the last dist.
I think it's the node cache problem. So I want to clear the cache of node by creating a sandbox.
Create a new sandbox to execute renderpage when replacing dist.But it didn't work out.

**Comment by brillout** [comment]
I think I know what's going on. Can you update to `vite-plugin-ssr@0.4.82` and report back?

**Comment by Kelier** [comment]
Dist1:
<img width="1213" alt="Welcome 110" src="https://user-images.githubusercontent.com/16694478/218712010-f8dc802f-685d-45d4-b6fc-18dcb67bacfc.png">

Dist2:
<img width="1176" alt="Welcome 110" src="https://user-images.githubusercontent.com/16694478/218712047-33209815-de61-42ed-b2d5-ee7b43f2ae78.png">

How to properly repair the file index？

**Reply by brillout** [reply]
Change `const DEBUG = false;` to `const DEBUG = true;` at `node_modules/@brillout/vite-plugin-import-build/dist/loadServerBuild/debugLogs.js:5` and report back the debug logs.

**Comment by kemengwang** [comment]
![image](https://user-images.githubusercontent.com/44251035/218712951-2b57e030-e8a5-4b05-bf33-3d41fde0cad8.png)

i has updated to vite-plugin-ssr@0.4.82.But it didn't work out.

**Reply by kemengwang** [reply]
@brillout

**Reply by brillout** [reply]
Change `const DEBUG = false;` to `const DEBUG = true;` at `node_modules/@brillout/vite-plugin-import-build/dist/loadServerBuild/debugLogs.js:5` and report back the debug logs.

**Reply by kemengwang** [reply]
initial dist(normal):
![image](https://user-images.githubusercontent.com/44251035/218723627-b538bb60-737e-4be4-bbc2-1d063f2d2d01.png)


replace dist 
![image](https://user-images.githubusercontent.com/44251035/218723901-deb27285-a894-4d22-8780-45ac764d4e48.png)

**Reply by kemengwang** [reply]
The debug message only appears  at initial dist.

**Reply by kemengwang** [reply]
@brillout

**Reply by brillout** [reply]
I don't see any problems.

I'll need a minimal reproduction.

**Reply by brillout** [reply]
> publishing and directly replace dist products without restarting the service, page access is still the result of the last dist.

You need to restart the Node.js server. That's unrelated to VPS.

**Reply by brillout** [reply]
See https://brillout.github.io/rules/.

**Reply by Kelier** [reply]
> > publishing and directly replace dist products without restarting the service, page access is still the result of the last dist.
> 
> You need to restart the Node.js server. That's unrelated to VPS.

How to return the server-side pageHtml in the new dist directory without restarting the server?

```
<div id="page-view">welcome 111</div>  => <div id="page-view">welcome 222</div>
```

**Reply by kemengwang** [reply]
Can I use VPS to update dist files without restarting the service? @brillout

**Reply by kemengwang** [reply]
> See https://brillout.github.io/rules/.

ok thanks
